mozart/services/api_v01.py

```diff
@@ -1190,12 +1211,16 @@ def post(self):
 
     def put(self):  # TODO: add user role and permissions
         request_data = request.json or request.form
-
-        _id = request_data.get('id')
-        if not _id:
+        _id = None
+        _rule_name = None
+        if "id" in request_data:
```

Comment:
not sure why i originally added `result` in the return data, it should be `success`

i think we can also do something like this

```
_id = request_data.get('id', None)
_rule_name = request_data.get("rule_name", None)

if not _id and not _rule_name:
    return {
        "success": False,
        "message": "message": "Must specify id or rule_name in the request"
    }
```